instrumentation/spring/spring-batch-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/batch/v3_0/jsr/TestItemWriter.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.jsr;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import javax.batch.api.chunk.ItemWriter;
+import org.codehaus.groovy.runtime.DefaultGroovyMethods;
+
+public class TestItemWriter implements ItemWriter {
+  @Override
+  public void open(Serializable checkpoint) {}
+
+  @Override
+  public void close() {}
+
+  @Override
+  public void writeItems(List<Object> items) {
+    for (Object item : items) {
+      this.items.add(DefaultGroovyMethods.asType(item, Integer.class));
+    }
+  }
+
+  @Override
+  public Serializable checkpointInfo() {
+    return null;
+  }
+
+  private final List<Integer> items = new ArrayList<>();
```

Comment:
for many of these classes the fields should [go at the top](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#ordering-of-class-contents)